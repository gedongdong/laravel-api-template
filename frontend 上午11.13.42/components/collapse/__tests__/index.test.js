import { mount } from '@vue/test-utils';
import Collapse from '..';

describe('Collapse', () => {
  it('should support remove expandIcon', () => {
    const wrapper = mount({
      render() {
        return (
          <Collapse expandIcon={() => null}>
            <Collapse.Panel header="header" />
          </Collapse>
        );
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
