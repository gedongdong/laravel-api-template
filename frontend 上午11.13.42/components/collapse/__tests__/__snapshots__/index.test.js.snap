// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collapse should support remove expandIcon 1`] = `
<div class="ant-collapse ant-collapse-icon-position-left">
  <div class="ant-collapse-item">
    <div class="ant-collapse-header" role="button" tabindex="0" aria-expanded="false">
      <div>
        <!---->
      </div>header
      <!---->
    </div>
    <!---->
  </div>
</div>
`;
